src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,14 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(r -> r.split(","))
+                .flatMap(Arrays::stream)
```

Comment:
![image](https://user-images.githubusercontent.com/80981426/215801763-d651e686-f5ee-4d72-939e-afb665e83515.png)
```suggestion
                .flatMap(string -> Arrays.stream(string.split(",")))
```
And what does this `r` stands for? let's try to use a more informative variable naming